index.d.ts

```diff
@@ -0,0 +1,49 @@
+import build = require('./lib/build');
+import validate = require('./lib/validate');
+import merge = require('./lib/merge');
+import shortcuts = require('./lib/shortcuts');
```

Comment:
It is nice that we can `require` our JavaScript modules and TypeScript/IntelliSense will use all the JSDoc annotations defined in those files/modules!